docker-compose.yml

```diff
@@ -40,6 +40,33 @@ services:
     volumes:
       - .:/app
 
+  wordpress_e2e_test:
+    image: wordpress
+    ports:
+      - 8889:80
+    environment:
+      WORDPRESS_DB_NAME: wordpress_e2e_test
+      WORDPRESS_DB_PASSWORD: example
+      ABSPATH: /usr/src/wordpress/
+    volumes:
+      - wordpress_e2e_test:/var/www/html
+      - .:/var/www/html/wp-content/plugins/gutenberg
+      - ./test/e2e/test-plugins:/var/www/html/wp-content/plugins/gutenberg-test-plugins
+      - ./test/e2e/test-mu-plugins:/var/www/html/wp-content/mu-plugins
+
+  mysql_e2e_test:
```

Comment:
This seems like overkill, but I couldn't find a(n easy) way to create two separate databases we could use in the same container from different WordPress images. I'd like to just have a `wordpress` and `wordpress_e2e_test` database in the same container, so if we can easily do that, let me know.